Document games (#518)
discord-userdoccers/discord-userdoccers: pages/resources/application.mdx

}
```

### Game Data Overrides Object

###### Game Data Overrides Structure

| Field                | Type                                                                 | Description                                                                                  |

> | -------------------- | -------------------------------------------------------------------- | -------------------------------------------------------------------------------------------- |3

Reviewer: ```suggestion
| -------------------- | -------------------------------------------------------------------- | -------------------------------------------------------------------------------------------- |
```

| 2     | PLAYSTATION_5 | PlayStation 5 |
| 3     | PLAYSTATION_4 | PlayStation 4 |

> ### Detectable Application Object

Reviewer: move to game resources

###### Linked Application Type

| Value | Name     | Description                          |
| ----- | -------- | ------------------------------------ |
| 1     | LINKED   | Application is linked                |
| 2     | OFFICIAL | Application is official              |
| 3     | NVIDIA   | Application is on NVIDIA GeForce NOW |

###### Example Detectable Application

```json
{
  "aliases": ["PUBG: BATTLEGROUNDS", "PUBG"],
  "executables": [
    {
      "is_launcher": false,
      "name": "win64/tslgame_be.exe",
      "os": "win32"
    },
    {
      "is_launcher": false,
      "name": "win64/tslgame.exe",
      "os": "win32"
    },
    {
      "is_launcher": false,
      "name": "tslgame.exe",
      "os": "win32"
    },
    {
      "is_launcher": false,
      "name": "win64/tslgame_uc.exe",
      "os": "win32"
    },
    {
      "is_launcher": false,
      "name": "tslgame_be.exe",
      "os": "win32"
    }
  ],
  "hook": true,
  "id": "356873622985506820",
  "name": "PLAYERUNKNOWN'S BATTLEGROUNDS",
  "overlay": true,
  "overlay_compatibility_hook": true,
  "overlay_methods": null,
  "overlay_warn": false,
  "themes": ["Action", "Warfare"]
}
```

> ### Game Claim Object

Reviewer: if we're gonna have a game resource, we may as well move game claims and their eps there too

###### Game Claim Structure

| Field             | Type                                                        | Description                                    |

> | ----------------- | ----------------------------------------------------------- | ---------------------------------------------- |2

Reviewer: ```suggestion
| ----------------- | ----------------------------------------------------------- | ---------------------------------------------- |
```

| game_id           | snowflake                                                   | The ID of the unclaimed game application       |
| application_id    | snowflake                                                   | The ID of the application                      |

> | claim_status      | string                                                      | The [status of game claim](#game-claim-status) |

Reviewer: ```suggestion
| claim_status      | string                                                      | The [game claim status](#game-claim-status) |
```

| ---------- | --------------------------------------------------------- | -------------------------------- |
| attachment | [attachment](/resources/message#attachment-object) object | The created ephemeral attachment |

> <RouteHeader method="GET" url="/applications/detectable" deprecated unauthenticated>

Reviewer: this endpoint was renamed to `/games/detectable`, it's just an alias now (the responses are identical), so we can just remove it

  Get Detectable Applications
</RouteHeader>

Returns a list of [detectable application](#detectable-application-object) objects representing games that can be detected by Discord for rich presence.

<RouteHeader method="GET" url="/applications/non-games/detectable" unauthenticated>
  Get Detectable Non Game Applications

> </RouteHeader>

Reviewer: should also prolly go in game resource

| Field     | Type   | Description                                                                                       |
| --------- | ------ | ------------------------------------------------------------------------------------------------- |
| username? | string | The username (max 1024)                                                                           |

> | metadata? | object | Custom metadata for the user application profile (max 25 keys, 1024 characters per key and value) |=

Reviewer: ```suggestion
| metadata? | object | Custom metadata for the user application profile (max 25 keys, 1024 characters per key and value) |
```

  Get Application Game Claims

> </RouteHeader>

Reviewer: game claim endpoints should be moved along with resource

| --------- | ------ | ------------------------------------------------------------------------------------------------- |
| username? | string | The username (max 1024)                                                                           |
| metadata? | object | Custom metadata for the user application profile (max 25 keys, 1024 characters per key and value) |

<RouteHeader method="GET" url="/applications/{application.id}/game-claims">
  Get Application Game Claims
</RouteHeader>

> Returns a list of [game clain](#game-claim-object) objects for the given application ID.

Reviewer: ```suggestion
Returns a list of [game claim](#game-claim-object) objects for the given application ID.
```

> <RouteHeader method="GET" url="/applications/{application.id}/game-claims/{game.id}">

Reviewer: ```suggestion
<RouteHeader method="GET" url="/applications/{application.id}/game-claims/{game_claim.id}">
```
change other routes as well, and add `game_claim` to the lib/endpoints file so they hyperlink

  Get Application Game Claim
</RouteHeader>

Returns a [game claim](#game-claim-object) object for the given application ID.

<RouteHeader method="PUT" url="/applications/{application.id}/game-claims/{game.id}">
  Create Application Game Claim
</RouteHeader>

> Creates a game claim for the given application ID. Returns a [game claim](#game-claim-object) object on success. May fire an [Guild Official Game Applications Update](/gateway/gateway-events#guild-official-game-applications-update) Gateway event.

Reviewer: ```suggestion
Creates a game claim for the given application ID. Returns a [game claim](#game-claim-object) object on success. May fire a [Guild Official Game Applications Update](/gateway/gateway-events#guild-official-game-applications-update) Gateway event.
```

###### Query String Params

| Field                     | Type    | Description                                                |
| ------------------------- | ------- | ---------------------------------------------------------- |
| verification_code?        | string  | The verification code (max 8 characters)                   |
| reset_verification?       | boolean | Whether to reset verification (default false)              |

> | resend_verification_code? | boolean | Whether to resend verification code (default false)        |

Reviewer: ```suggestion
| resend_verification_code? | boolean | Whether to resend the verification code (default false)        |
```

> | full_name?                | string  | The full legal name (max 100 characters)                   |

Reviewer: ```suggestion
| full_name?                | string  | Full legal name of the applicant (max 100 characters)                   |
```

> | business_email?           | string  | The business email (max 320 characters)                    |

Reviewer: ```suggestion
| business_email?           | string  | Business email of the applicant (max 320 characters)                    |
```

> | business_role?            | string  | The business role of the current user (max 100 characters) |

Reviewer: ```suggestion
| business_role?            | string  | Business role of the applicant (max 100 characters) |
```
